packages/editor/src/components/char-count/index.js

```diff
@@ -0,0 +1,24 @@
+/**
+ * WordPress dependencies
+ */
+import { withSelect } from '@wordpress/data';
+import { _x } from '@wordpress/i18n';
+import { count as charCount } from '@wordpress/wordcount';
+
+function CharCount( { content } ) {
+	/*
+	 * Two options available for counting: 'characters_excluding_spaces' or 'characters_including_spaces'
+	 * Do not translate into your own language.
+	 */
+	const charCountType = _x( 'characters_including_spaces', 'Character count type. Do not translate literally!' );
```

Comment:
It would be good to export this, as well in `word-count`, to be used `table-of-contents` to compare.